XenModel/XenAPI-Extensions/Address.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace XenAPI
+{
+    public static class Address

```

Comment:
On second thought, do we need a dedicated class for this? It would make sense if we could gather here other methods dealing with IPs, but if it's only one method, we could perhaps add it to XenModel\Utils\Helper.cs (which is normally where we look for utility methods)?